remix-astwalker/src/sourceMappings.ts

```diff
@@ -1,28 +1,60 @@
 import { isAstNode, AstWalker } from './astWalker';
-import { AstNode, Location } from "./types";
+import { AstNode, LineColPosition, LineColRange, Location } from "./types";
+const util = require("remix-lib").util;
```

Comment:
could we use here the ` import ... from ` ?

Reply:
Sure. Note though this was not as easy as just changing that line...

`remix-lib` is not in typescript so there are not types for that and I could not find a `@types/remix-lib`. Rather than undertake writing such a module, I opted for starting one of the one function I need in `remix-astWalker`. I am new to this stuff, so I don't know if this was done properly.